Storage/SQL/FaviconsTable.swift

```diff
@@ -72,7 +72,7 @@ class FaviconsTable<T>: GenericTable<Favicon> {
         if (cursor.count != 1) {
             return nil
         }
-        return (cursor[0] as Favicon).id
+        return (cursor[0] as? Favicon)?.id
```

Comment:
Yeah. A huge amount of the fixes in this PR were casts pulling stuff out of cursors :/